templates/monorepo/apps/azure-functions-api/jest.config.js

```diff
@@ -0,0 +1,7 @@
+require("dotenv").config({ path: "env.example" });
```

Comment:
Jest is on hold in our techradar

Reply:
Fixed in [0b26bdc](https://github.com/pagopa/dx-typescript/pull/4/commits/0b26bdc93a917b81d02281042d34e99fac34a11c)